### ISSUE TITLE
Can't install tifffile with requirements.txt into newly created virtual environment.

### ISSUE THREAD

**Issue by zakattak** [blink1073/tifffile#25]
tifffile's setup.py uses numpy.get_include() which causes my pip install to fail.

My requirements.txt is as follows:
numpy==1.11.1
Pillow==3.3.1
scipy==0.18.1
tifffile==0.10.0
matplotlib==2.0.0
scikit-image==0.13.0

When I `pip install -r requirements.txt` in a newly created virtual environment, the install fails because pip install doesn't install each line at a time. It checks to see that all dependencies can be installed before it begins any. Since numpy has not been installed yet, it fails on the tifffile setup.py, because it uses numpy.get_include(). 

I don't want to have to split up my requirements.txt and install tifffile after numpy. It would be best if all python packages only depended on setuptools in their setup.py.

Here's the full output of the error:
```
venv-setup:
     [exec] New python executable in /allen/aics/apps/prod/jenkins/jenkins-01/venv/aicsimage/bin/python
     [exec] Installing setuptools, pip, wheel...done.
     [exec] Collecting numpy==1.11.1 (from -r /allen/aics/apps/prod/jenkins/jenkins-01/workspace/aicsimagepipeline/requirements.txt (line 1))
     [exec]   Downloading https://artifactory.corp.alleninstitute.org/artifactory/api/pypi/pypi-virtual/packages/18/eb/707897ab7c8ad15d0f3c53e971ed8dfb64897ece8d19c64c388f44895572/numpy-1.11.1-cp27-cp27mu-manylinux1_x86_64.whl (15.3MB)
     [exec] Collecting Pillow==3.3.1 (from -r /allen/aics/apps/prod/jenkins/jenkins-01/workspace/aicsimagepipeline/requirements.txt (line 2))
     [exec]   Downloading https://artifactory.corp.alleninstitute.org/artifactory/api/pypi/pypi-virtual/packages/e2/41/807b1628be3ef056f53519d4a96ff38fa5d12b46d3af7ddf8982461b1da8/Pillow-3.3.1-cp27-cp27mu-manylinux1_x86_64.whl (5.6MB)
     [exec] Collecting scipy==0.18.1 (from -r /allen/aics/apps/prod/jenkins/jenkins-01/workspace/aicsimagepipeline/requirements.txt (line 3))
     [exec]   Downloading https://artifactory.corp.alleninstitute.org/artifactory/api/pypi/pypi-virtual/packages/13/cb/8e74d28e1519b34636e4d985d49d01c23778064e01eb102914f844cd6051/scipy-0.18.1-cp27-cp27mu-manylinux1_x86_64.whl (40.3MB)
     [exec] Collecting tifffile==0.10.0 (from -r /allen/aics/apps/prod/jenkins/jenkins-01/workspace/aicsimagepipeline/requirements.txt (line 4))
     [exec]   Downloading https://artifactory.corp.alleninstitute.org/artifactory/api/pypi/pypi-virtual/packages/6b/bc/a471bb7d9119a1751466649a23a387e9cd38ee8d4227b5a075916e9eea81/tifffile-0.10.0.tar.gz (71kB)
     [exec]     Complete output from command python setup.py egg_info:
     [exec]     Traceback (most recent call last):
     [exec]       File "<string>", line 1, in <module>
     [exec]       File "/tmp/pip-build-Lsg575/tifffile/setup.py", line 7, in <module>
     [exec]         import numpy
     [exec]     ImportError: No module named numpy
     [exec]     
     [exec]     ----------------------------------------
     [exec] Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Lsg575/tifffile/
```

**Comment by zakattak** [comment]
Also, see: https://github.com/pypa/pip/issues/25

**Comment by blink1073** [comment]
We need the headers files from numpy in order to build the extension module.  Do you require the speedups from the extension module?

**Comment by zakattak** [comment]
No, I don't believe that I require them. Is there a way to use tifffile without the extension module? I wasn't aware.

**Comment by blink1073** [comment]
Yep, see #26.

**Comment by zakattak** [comment]
Thanks!

**Comment by blink1073** [comment]
No problem.  Released as 0.12.1.
